Module/data/clearData.js

```diff
@@ -0,0 +1,14 @@
+const mongodb = require('mongodb');
+
+const uri = 'mongodb://localhost/carebnb';
+// Connect to Database
+mongodb.MongoClient.connect(uri, (connectionErr, client) => {
+  if (connectionErr) throw connectionErr;
+  // Clear Database
+  client.db('carebnb').dropDatabase((dropErr) => {
```

Comment:
Would it make sense to have one file that drops the database every time it is run and then repopulates it? Essentially combining clearData.js and addData.js.

Reply:
Thank you, yeah i changed it!